changelog/1.23.1.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "1.23.1"
+description: "Released on 10/11/2021"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.23.1.
+
+### Features ✨
+
+There are no new features in 1.23.1.
+
+### Bug fixes 🐛
+
+- web: fixed issue with Coder not displaying an error when users do not have
+  permission to log in via OIDC.
+- web: fixed issue where large output would cause web terminals to disconnect.
+- web: fixed issue with Intercom not loading for hosted beta users.
+- web: fixed issue with RStudio not launching.
+- web: fixed issue with password max length validation being too narrow for
+  select password types (password length limit for image registries have been
+  updated to 32 KiB).
```

Comment:
only updated registry AFAIK so maybe change the wording to not be "select password types" but just "registries"